src/main/java/de/rabea/game/GameType.java

```diff
@@ -48,7 +47,8 @@ private HumanPlayer createNewHumanOpponent() {
         return new HumanPlayer(userInterface, O);
     }
 
-    private ComputerPlayer createNewComputerPlayer() {
-        return new ComputerPlayer(new RandomNumberCalculator(), O);
+    private UnbeatableComputerPlayer createNewComputerPlayer() {
+//        return new ComputerPlayer(new RandomNumberCalculator(), O);
+        return new UnbeatableComputerPlayer(O);
```

Comment:
I'll sort this bit out in the next PR for which I will need to create a menu for all game options. I used the `UnbeatableComputerPlayer` here for now, just so I can play against it.